### DISCUSSION TITLE
Determining edge weights

### DISCUSSION THREAD

**Discussion by cernaksimon** [gephi/gephi#2599] [Q&A]
Hello all

i am dealing with a social network dataset where i am looking at scientific paper authors, from Google Scholar, and their collaborations. So i have a spreadsheet i am importing and it looks like this. 

<img width="1347" alt="Screenshot 2022-08-31 at 15 52 55" src="https://user-images.githubusercontent.com/45256612/187695431-7f692650-9d5e-473a-8a52-be7925ccb9d1.png">

Basically i would like to determine which authors are collaborating the most, but i feel like my values for edge weight are not realistic. Im fairly certain that (almost) every pair of authors did not collaborated this many times, as displayed in data lab:

![Screenshot 2022-08-31 at 15 55 41](https://user-images.githubusercontent.com/45256612/187696424-84c0f6ef-f65c-4682-a3b0-6747ade237f9.png)

Also it always just says one article title so there is really no way to know if the collaborated on "17" more.

So im wondering if im choosing a wrong merge strategy when importing my spreadsheet. I usually used the default settings:
![image](https://user-images.githubusercontent.com/45256612/187696939-22b93ec1-b2bf-4fdc-b251-7f2422424021.png)

Basically asking for any thoughts on this, as i am lost how to present who is collaborating with who the most.

Thanks:)

**Comment by mbastian** [comment]
Hi @cernaksimon, interesting dataset ;)

I think your approach is sound. I see that you've selected the "Sum" edge merge strategy so that should work as you describe it. The weight represent the number of papers they authored together. One thing that might not be intuititve is that merging edges also means edges get dropped from your dataset. The weight increments itself but if you have 20 edges with the same Source - Target in your dataset, 19 of those edges will not appear in Gephi. Meaning, in your case the `article_title` column is only a subset.

In the future, we could support merging attributes too but that's not the case. That would give you an array of articles.

If you do find the weight to be wrong as you suggest, I suggest that you have a closer look at your dataset. Maybe some duplicate edges. Also, it's possible the edge is counted twice in each direction as your network is undirected.
